hasura/metadata/databases/intuition/tables/public_cached_image.yaml

```diff
@@ -1,6 +1,6 @@
 table:
   name: cached_image
-  schema: public
+  schema: cached_images
```

Comment:
Boiko to look into this -- I needed to change this in order to compile and run locally